main/lsp/test/lsp_preprocessor_test.cc

```diff
@@ -125,116 +106,6 @@ class CountingTask final : public core::lsp::Task {
 
 } // namespace
 
-TEST(TimeTravelingGlobalState, ComesBefore) { // NOLINT
```

Comment:
I deleted all of the tests involving `TimeTravelingGlobalState` and fast path decisions, since the preprocessor is no longer involved with those matters.